pkg/apis/admissionregistration/v1/doc.go

```diff
@@ -0,0 +1,28 @@
+/*
+Copyright 2020 Authors of Arktos.
+
```

Comment:
This file is missing kubernetes copyright. Is this correct?

Reply:
This is by design for current arktos_copyright.sh. all new added files only has arktos copyrights if we didn't  add new added files to hack/arktos_copyright_copied_k8s_files.